meta-iotqa/lib/oeqa/runtime/peripherals/mraa/mraa_commands.py

```diff
@@ -0,0 +1,77 @@
+import os
+from oeqa.oetest import oeRuntimeTest
+'''
+This test suit tests functioning of four options of  of mraa-gpio command
+mraa-gpio version
+mraa-gpio list
+mraa-gpio set <pin> <value>
+mraagpio get <pin>
+
+'''
+class MraaGPIOTest(oeRuntimeTest):
+    #source locations
+    mraa_cmd_test_python_script = 'mraa_gpio_commands_test.py'
+    mraa_get_test_script = 'mraa_get_test.sh'
+    mraa_set_test_script = 'mraa_set_test.sh'
+    mraa_list_test_script = 'mraa_list_test.sh'
+    mraa_version_test_script = 'mraa_version_test.sh'
+
+
+    # target locations
+    mraa_cmd_test_python_script_target = '/tmp/%s' % mraa_cmd_test_python_script
+    mraa_get_test_script_target = '/tmp/%s' % mraa_get_test_script
+    mraa_set_test_script_target = '/tmp/%s' % mraa_set_test_script
+    mraa_list_test_script_target = '/tmp/%s' % mraa_list_test_script
+    mraa_version_test_script_target = '/tmp/%s' % mraa_version_test_script
+
+
+    def SetUp(self):
```

Comment:
should this be called setUp(self) ?

seems it is not called prior to testong, causing lack of files in /tmp